code/modules/cargo/randomstock.dm

```diff
@@ -1096,7 +1096,7 @@ var/list/global/random_stock_large = list(
 		if("chempack")
 			var/total = rand(2,6)
 			var/list/chems = SSchemistry.chemical_reagents.Copy()
-			var/list/exclusion = list("drink", "reagent", "adminordrazine", "beer2", "azoth", "elixir_life", "liquid_fire", "philosopher_stone", "undead_ichor")
+			var/list/exclusion = list("drink", "reagent", "adminordrazine", "beer2", "azoth", "elixir_life", "liquid_fire", "philosopher_stone", "undead_ichor", "love", "shapesand", "usolve", "sglue", "black_matter")
```

Comment:
This really should be made a variable on the reagent's end instead of an ever-expanding list.

Reply:
Perhaps in another time, but this should work for now.